### DISCUSSION TITLE
Restore Model in a Different Script

### DISCUSSION THREAD

**Discussion by ppandit95** [lululxvi/deepxde#1057] [Q&A]
Dear @lululxvi 

I was wondering if it is possible to restore the previously trained model in a different script so that I can run only the restored model training to the point where I get satisfactory solution to my problem.

Many Thanks
Pushkar

**Comment by lululxvi** [comment]
See FAQ.

**Comment by ppandit95** [comment]
Thanks @lulu i did manage to restore the model from one of the previous
discussions .
